docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/global-permissions.md

```diff
@@ -171,6 +102,112 @@ For details on which Kubernetes resources correspond to each global permission,
 
 :::
 
+### Custom GlobalRoles
+
+Users may wish to create custom GlobalRoles as a way of satisfying a use case not directly addressed by our built-in GlobalRoles. 
+
+Users can create a custom GlobalRole using the UI or one of the supported automation methods. The rules that can specified are of the same type as the rules used by upstream roles and clusterRoles. 
```

Comment:
I left it a bit broad since you can use [terraform](https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/global_role) as well. I can make the swap if we are ok with omitting that from our docs though.

Reply:
I've adjusted the language a bit here, but I don't think it's a good idea to attempt to enumerate all supported automation methods here. Let me know if you'd like to discuss further.